doc/usage.rst

```diff
@@ -46,11 +46,11 @@ table:
 
 .. image:: Approved.png
 
-This will bring you to the page to vote.
+This will bring you to the page where you can vote.
 
-In front of you, in the middle of the page, on a white background are
-presented all the candidates for the election you selected.
-By hovering your mouse over the image, you can see two icons appearing:
+In front of you, in the middle of the page, all the candidates for the election you selected
+are presented on a white background. By hovering your mouse over the image, you will be able to
```

Comment:
@pypingou I dont think so, but since it was already there, I only paraphrased the text. Do you think it is not needed?